pkg/cmd/roachprod-microbench/main.go

```diff
@@ -191,6 +192,19 @@ func makeExportCommand() *cobra.Command {
 	return cmd
 }
 
+func makeCompressCommand() *cobra.Command {
+	runCmdFunc := func(cmd *cobra.Command, commandLine []string) error {
+		return compress(cmd.OutOrStdout(), cmd.InOrStdin())
```

Comment:
Was there a reason this was forced to be stdin and stdout? The automation script needs to redirect stdout to a file. If anyone attempts to use this command locally without knowing about this, it will print random bytes to their terminal, which is never pleasant. Maybe be worth it to accept input and output files.

Reply:
The main reason for `stdin` is I want to have this pipe directly from `tar` to avoid an interim large file. But as for `stdout` could likely be set to point to a file, but I'd like to keep it simple, since this will most likely ever be used by CI.